application/src/main/java/org/togetherjava/tjbot/commands/code/CodeMessageAutoDetection.java

```diff
@@ -22,12 +24,12 @@ public final class CodeMessageAutoDetection extends MessageReceiverAdapter {
     private static final long MINIMUM_LINES_OF_CODE = 3;
 
     private final CodeMessageHandler codeMessageHandler;
-
     private final Predicate<String> isHelpForumName;
+    private final Predicate<String> isExcludedRolePattern;
```

Comment:
```suggestion
    private final Predicate<String> isExcludedRole;
```
Since it's not a regex pattern anymore, but a `Predicate`. ^^

Reply:
m lord ^^